java/code/src/com/suse/manager/webui/services/impl/SaltSSHService.java

```diff
@@ -588,8 +588,7 @@ private Optional<Map<String, Object>> minionOpts(String minionId,
     private Optional<String> remotePortForwarding(List<String> proxyPath,
                                                   String sshContactMethod) {
         if (ContactMethodUtil.SSH_PUSH_TUNNEL.equals(sshContactMethod)) {
-            return Optional.of(getSshPushRemotePort() + ":" +
-                    ConfigDefaults.get().getJavaHostname() + ":" + SSL_PORT);
```

Comment:
I wonder how this worked even before with chain of proxies. This would always point to the server, wouldn't it? So not a local redirection. Do we have some other postprocessing step? e.g. translating the server to the proxy fqdn?

Reply:
I think `ConfigDefaults.get().getJavaHostname()` is not what 4.3 has. This was already tried to be fixed.
But I think we did not test for a long time chain proxies